### PR TITLE
Tweak WrongModuleTypeNotificationProvider (#466).

### PR DESCRIPTION
**Pull request by pq**
Adds a new action to `flutter.yaml` files that allows you to enable Flutter support.

screen shot 2016-12-05 at 3 59 29 pm

Partially address: #466.

/cc @devoncarew

### CODE REVIEW

File: src/io/flutter/inspections/FlutterYamlNotificationProvider.java [hunk 1]
```diff
@@ -64,6 +79,28 @@ public EditorNotificationPanel createNotificationPanel(@NotNull VirtualFile file
...
+    myModule = module;
+    myProject = project;
+  }
+
+  @Override
+  public void run() {
+
+    myModule.setOption(Module.ELEMENT_TYPE, FlutterModuleType.getInstance().getId());
+
+    if (DartSdk.getDartSdk(myProject) != null && !DartSdkGlobalLibUtil.isDartSdkEnabled(myModule)) {
```
Comment (devoncarew) [inline src/io/flutter/inspections/FlutterYamlNotificationProvider.java]: How will this behave if there's no setting for a dart sdk? Perhaps only show the editor notification if there's not a flutter nature for the module, and we know that we can fix the issue (there's a flutter sdk defined)?

**Comment by devoncarew** [comment]
lgtm; one question about when best to show this, and how we should behave when there's no flutter sdk installed

**Comment by alexander-doroshko** [comment]
Doesn't it duplicate the idea of `WrongModuleTypeNotificationProvider`?

**Comment by pq** [comment]
@alexander-doroshko : it does!  Funny that I overlooked that.

I've removed the new inspection and tweaked the text of the existing one a bit.

screen shot 2016-12-06 at 6 55 11 am

**Comment by devoncarew** [comment]
The labels seem a bit long; word-smithing to follow in-line -

File: src/io/flutter/FlutterBundle.properties [hunk 2]
```diff
@@ -31,16 +29,14 @@ flutter.project.description=Flutter modules are used to build high-performance,
 flutter.sdk.browse.path.label=Select Flutter SDK Path
 flutter.sdk.is.not.configured=The Flutter SDK is not configured
 flutter.sdk.path.label=Flutter &SDK path:
+flutter.support.is.not.enabled.for.module.0=Flutter support is not enabled for the module ''{0}''
```
Comment (devoncarew) [inline src/io/flutter/FlutterBundle.properties]: remove `the` to make this slightly shorter?

**Comment by devoncarew** [comment]
And perhaps `'Change module type to Flutter'` and `Don't show again`?

File: src/io/flutter/FlutterBundle.properties (hunk 2 above)
Comment (pq) [inline src/io/flutter/FlutterBundle.properties]: Sure!

**Comment by devoncarew** [comment]
still lgtm; thanks for the wording update

**Pull request merged**